### DISCUSSION TITLE
Choosing tar_plan saving format

### DISCUSSION THREAD

**Discussion by cbellard** [ropensci/tarchetypes#81] [Help]
Is there a way to select a format (eg qs) other than the default format for saving every objects created with a tar_plan call ?
I think I could modify the hidden function tar_plan_parse_command which call tar_target but there is surely a better way.

**Comment by wlandau** [comment]
You could use `tar_option_set()` in between calls to `tar_plan()`.

```r
tar_option_set(format = "qs")
targets1 <- tar_plan(qs_target_1 = cmd1(), qs_target_2 = cmd2()) # all use qs format
tar_option_set(format = "feather")
targets2 <- tar_plan(feather_target_1 = cmd3(), feather_target_2 = cmd4()) # all use feather format
```
